templates/404.html

```diff
@@ -10,13 +14,15 @@
             <div class="errorbox">
                 <div class="errorcontent">
                     {% if status_code == 405 %}
-                    <h1 class="lead">Method not allowed (405)</h1>
+                    <h1 class="lead">{{ _("Method not allowed (405)") }}</h1>
                     {% else %}
-                    <h1 class="lead">Page not found (404)</h1>
+                    <h1 class="lead">{{ _("Page not found (404)") }}</h1>
                     {% endif %}
                     <p>
+                        {% trans %}
                         If this error is unexpected, you can
                         <a href="mailto:{{ support_email }}">contact support</a>.
+                        {% endtrans %}
                     </p>
```

Comment:
Noted that this template wasn't translated, so added that as well.

Reply:
Looks correct, though better would be to have the HTML be outside the translation block like we do in some email templates.